.travis.yml

```diff
@@ -92,9 +84,8 @@ jobs:
         - |
           release_version=$(echo "${TRAVIS_TAG}" | sed 's/^release-//') || travis_terminate 1
           git push origin :"${TRAVIS_TAG}"
-        # Prepare and push release commits. '[skip ci]' ensures Travis doesn't build these commits.
-        # These commits push immediately as time uploading deployments adds git conflict risk.
-        - ${mvn_release} -DreleaseVersion=${release_version} -Darguments=-DskipTests release:prepare
+        # Prepare and push release commits and the version tag (N.N.N), which triggers deployment.
+        - ./mvnw --batch-mode -nsu -DreleaseVersion=${release_version} -Darguments=-DskipTests release:prepare
```

Comment:
release profile isn't used in release:prepare, only release:perform (which we never use)

release:prepare is almost the same as running set-version